BibBuddy/app/src/main/java/de/bibbuddy/NotesFragment.java

```diff
@@ -115,7 +119,7 @@ private void handleDeleteNote() {
           getString(R.string.delete_note_message) + " " + getString(R.string.delete_warning));
     }
 
-    alertDeleteBookNote.setNegativeButton(R.string.back, (dialog, which) -> deselectNoteItems());
+    alertDeleteBookNote.setNegativeButton(R.string.back, (dialog, which) -> {});
```

Comment:
```suggestion
    alertDeleteBookNote.setNegativeButton(R.string.back, (dialog, which) -> { deselectNoteItems()});
```
I think, we should keep the deselection on canceling the deletion to make the user able to deselect multiple items at once - and deleting is the only available option that is affected by selected items, isn't it?

Reply:
I disagree here.
1) Deselecting multiple items at once works with the "back" button in any list fragment anyways.
2) Consistent behavior is important across all lists. In other lists, we have options of "delete" and "change". If a user now selects a list item and selects "delete" that may be an error, if he/she actually wanted to change. Pressing "zurück" now would deselect the item and he/she would have to select it one more time.

Reply:
Ah, I see. Keep it like that then :)

Reply:
Alright!